cli/index.ts

```diff
@@ -213,6 +213,11 @@ const cli = async (args: CommandLineArgs) => {
   if (clone.code !== 0) {
     console.log("Git clone failed. Exiting...");
   }
+  console.log("Removing .git ...");
+  const removeGit = shell.exec("rm -rf starter-code-v2/.git");
```

Comment:
Not a big issue since there's an error check for the `rm` already but we should actually exit early if the git clone failed

Reply:
I see that this was addressed in your other PR, feel free to merge after rebasing when #10 gets merged